src/EnergyPlus/ReportCoilSelection.hh

```diff
@@ -470,7 +493,7 @@ private: // methods
 	);
 
 
-private: // data
+public: // data
```

Comment:
@Myoldmopar @EnergyArchmage I couldn't get the unit tests to work without making this public. Is there a better way?

Reply:
https://stackoverflow.com/questions/2396370/how-to-make-google-test-classes-friends-with-my-classes

I think make it public.  Or better yet make your `class` a `struct` and just make things private that are definitely not needed anywhere else even in unit tests.